run-checks

```diff
@@ -116,6 +116,8 @@ missing_interface_spread_test() {
     done
 }
 
+echo broken
```

Comment:
This looks like a leftover?

Reply:
this is part of the commit with a bad commit author to demonstrate that the check works

Reply:
alright this is gone now, see the other comment for an example bad commit which causes the workflow to fail